helm-charts/interoperator/templates/operator_api_app.yaml

```diff
@@ -0,0 +1,102 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    service: {{ .Release.Name }}-op-apis-service
+  name: {{ .Release.Name }}-op-apis-service
+  namespace: {{ .Release.Namespace }}
+spec:
+  type: {{ .Values.operator_apis.service.type }}
+  ports:
+  - name: http
+    port: 80
+    targetPort: op-apis-port
+  - name: https
+    port: 443
+    targetPort: op-apis-port
+  selector:
+    app: {{ .Release.Name }}-op-apis-app
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  labels:
+    service: {{ .Release.Name }}-op-apis-service
+    {{- if .Values.cluster.ssl }}
+    garden.sapcloud.io/purpose: managed-cert
+    {{- end }}
+  name: {{ .Release.Name }}-op-apis-service-ingress
+  namespace: {{ .Release.Namespace }}
+spec:
+  rules:
+  - host: {{ .Values.operator_apis.host }}
+    http:
+      paths:
+      - backend:
+          serviceName: {{ .Release.Name }}-op-apis-service
+          servicePort: 80
+  {{- if .Values.operator_apis.ssl }} 
+  tls:
+  - hosts:
+    - {{ .Values.operator_apis.host }}
+    secretName: {{ .Values.operator_apis.sslsecret }}
+  {{- end }} 
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-op-apis-app
+  labels:
+    app: {{ .Release.Name }}-op-apis-app
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}-op-apis-app
+  template:
+    metadata:
+      labels:
+        app: {{ .Release.Name }}-op-apis-app
+      annotations:
+        rollme: {{ randAlphaNum 5 | quote }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 6 }}
+      {{- end }}
+      containers:
+      - name: {{ .Chart.Name }}
+        image: "{{ .Values.operator_apis.image.repository }}:{{ .Values.operator_apis.image.tag }}"
+        imagePullPolicy: {{ .Values.operator_apis.image.pullPolicy }}
+        ports:
+        - name: op-apis-port
+          containerPort: {{ .Values.operator_apis.port }}
+          protocol: TCP
+        env:
+        - name: OPERATOR_APIS_APP_USERNAME
+          value: {{ .Values.operator_apis.username }}
+        - name: OPERATOR_APIS_APP_PASSWORD
+          value: {{ .Values.operator_apis.password }}
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /
+            port: 9297
```

Comment:
This is not used from the values yaml. Also in the env OPERATOR_APIS_APP_PORT is not set from the values

Reply:
done